src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -219,7 +219,7 @@ private void AddOverviewPage(TableOfContentsNode toc)
 
             if (!string.IsNullOrEmpty(toc.Uid) || !string.IsNullOrEmpty(toc.Href))
             {
-                var overview = toc.CloneWithoutItems();
+                var overview = new TableOfContentsNode(toc) { Items = new List<SourceInfo<TableOfContentsNode>>() };
```

Comment:
```suggestion
                var overview = new TableOfContentsNode(toc);
```

Reply:
For the overview should have only one "Overview" item. Should make the `Items` empty firstly.

Reply:
Remove `Items = item.Items` in `TableOfContentsNode.cs` and fix all references of this constructor.